references/search.md

```diff
@@ -1,11 +1,107 @@
 # Search
 
-## Search in an index
+Meilisearch exposes 2 routes to perform searches:
+
+- A POST route: this is the preferred route when using API authentication, as it allows preflight request cache and better performances.
+- A GET route: the usage of this route is discouraged, unless you have good reason to do otherwise (specific caching abilities for example)
```

Comment:
```suggestion
- A GET route: the usage of this route is discouraged, unless you have good reason to do otherwise (specific caching abilities for example).
```